test/zdtm.py

```diff
@@ -2077,11 +2077,11 @@ def grep_errors(fname):
     print_next = False
     before = []
     with open(fname, errors='replace') as fd:
-        for l in fd:
-            before.append(l)
+        for line in fd:
```

Comment:
zdtm.py and Dockerfile.alpint changes should be in separate patches because these changes are independent.